tests/providers/test_ssn.py

```diff
@@ -423,6 +423,14 @@ def test_pt_BR_cpf(self):
         for _ in range(100):
             assert re.search(r'\d{3}\.\d{3}\.\d{3}-\d{2}', self.factory.cpf())
 
+    def test_pt_BR_rg(self):
+        for _ in range(100):
```

Comment:
I know there's plenty of existing test using this loop pattern, but we're trying to move away from it.

A better way is to use `factory.seed` to have the provider generate known values. Something like:
```python
    def test_pt_BR_rg(self):
        self.factory.seed(0)
        to_test = self.factory.rg()
        assert to_test.endswith('X')

        self.factory.seed(1)
        to_test = self.factory.rg()
        assert to_test[-1] not in ('X', 0)

        self.factory.seed(5)
        to_test = self.factory.rg()
        assert to_test.endswith('0')
```

Reply:
Ignore my comment. Testing with `factory.seed` is not currently that easy. I have a PR up that would make it easier, but it shouldn't be on you.